src/extension/content-script/liquid.js

```diff
@@ -66,9 +70,14 @@ async function init() {
         origin: getOriginData(),
       };
 
+      if (!account.hasMnemonic) {
+        messageWithOrigin.action = `public/liquid/providerOnboard`;
+      }
+
       const replyFunction = (response) => {
         if (ev.data.action === `${SCOPE}/enable`) {
           isEnabled = response.data?.enabled;
+          isOnboardingStarted = response.data?.onBoardStarted;
```

Comment:
`onBoardStarted` should be `onboardStarted`

Reply:
Can you use consistent naming for both?

Reply:
i was resolving promise with success reply and use this to handle next execution of function and avoid double popup of onboarding screen. 

for eg. nostr we call enable and call actual function. content script runs two times 

now we resolve promise with the error this is no more the case